core-model/src/main/kotlin/com/skydoves/pokedex/core/model/Pokemon.kt

```
@@ -35,5 +35,11 @@ data class Pokemon(
     return "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/" +
       "pokemon/other/official-artwork/$index.png"
   }
+
+  fun getIndex(): String {
+    val index = url.split("/".toRegex()).dropLast(1).last()
+    return String.format("#%03d", index.toInt())
```

detekt (1.23.8)

[warning] String.format("#%03d", index.toInt()) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)